server/jetstream_cluster.go

```diff
@@ -1667,15 +1670,23 @@ func (js *jetStream) metaSnapshot() ([]byte, error) {
 		return nil, err
 	}
 
-	// Track how long it took to compress the JSON
+	// Track how long it took to compress the JSON.
 	cstart := time.Now()
 	snap := s2.Encode(nil, b)
 	cend := time.Since(cstart)
+	took := time.Since(start)
 
-	if took := time.Since(start); took > time.Second {
+	if took > time.Second {
 		s.rateLimitFormatWarnf("Metalayer snapshot took %.3fs (streams: %d, consumers: %d, marshal: %.3fs, s2: %.3fs, uncompressed: %d, compressed: %d)",
 			took.Seconds(), nsa, nca, mend.Seconds(), cend.Seconds(), len(b), len(snap))
```

Comment:
@wallyqs Here in the log we show compressed and uncompressed size of the snapshot.
Does it make sense to expose it in jsz as well?

Reply:
Not sure if we should add those additional fields. That ties it a lot more to the current implementation. If for 2.14 we want to improve this and end up not needing to block inline and compress anymore, then these fields will not mean anything anymore.

I'd rather keep just the `last_time` and `last_duration`.